README.md

```diff
@@ -10,14 +10,207 @@ support new services is also a fairly simple process.
 
 ## Table of contents
 1. [Installation](#installation)
-2. [Documentation](http://docs.neuroinfo.org/lochness/en/latest/)
+2. [Quick setup from a template](#Setup from a template)
+3. [Documentation](http://docs.neuroinfo.org/lochness/en/latest/)
+
 
 ## Installation
-Just use `pip`
 
-```bash
-pip install lochness
+Just use ``pip`` ::
+
+    pip install lochness
+
+
+For most recent DPACC-lochness ::
+
+    pip install git+https://github.com/PREDICT-DPACC/lochness
+
+
+For debugging ::
+
+    cd ~
+    git clone https://github.com/PREDICT-DPACC/lochness
+    pip install -r ~/lochness/requirements.txt
+
+    export PATH=${PATH}:~/lochness/scripts  # add to ~/.bashrc
+    export PYTHONPATH=${PYTHONPATH}:~/lochness  # add to ~/.bashrc
+
+
+## Setup from a template
+
+### Creating the template
+Setting up lochness from scratch could be slightly confusing in the beginning.
+Try using the `lochness_create_template.py` to create a starting point.
+
+Create an example template to easily structure the lochness system
+
+```sh
+# PREDICT
```

Comment:
What's the difference b/w `PREDICT` and `PRESCIENT`?

Reply:
Different sources would be used for each network, eg) PREDICT uses REDCap whereas the PRESCIENT uses RPMS etc.

Reply:
Isn't the former called `ProNET`?

Reply:
My bad. `ProNET`